libs/core/src/lib/wizard/wizard.component.html

```diff
@@ -1,5 +1,15 @@
 <section class="fd-wizard">
     <ng-content select="fd-wizard-navigation"></ng-content>
-    <ng-container *ngTemplateOutlet="contentTemplate"></ng-container>
+    <div
+        class="fd-wizard-container-wrapper"
+        #wrapperContainer
+        fdScrollSpy
+        [trackedTags]="['div']"
+        (spyChange)="scrollSpyChange($event.children[0].id)"
```

Comment:
Let's just keep parameter as a $event and deal with rest on `scrollSpyChange`. I think it will be much cleaner for other users